### PR TITLE
add more log to show details if member with empty moniker

### PR DESCRIPTION
**Pull request by luyajun0205**
add more log to show details if member with empty moniker

### CODE REVIEW

File: ECMA2Yaml/IntellisenseFileGen/IntellisenseFileGenHelper.cs [hunk 1]
```diff
@@ -55,6 +57,10 @@ public static void Start(string[] args)
                 // TODO: log error
                 return;
             }
+            if (string.IsNullOrEmpty(opt.LogFilePath))
+            {
+                _logFile = "log.txt";
+            }
             (_repoRootFolder, _fallbackRepoRootFolder) = ECMALoader.GetRepoRootBySubPath(_xmlDataFolder);
             _fileAccessor = new FileAccessor(_repoRootFolder, _fallbackRepoRootFolder);
```
Comment (TianqiZhang) [inline ECMA2Yaml/IntellisenseFileGen/IntellisenseFileGenHelper.cs:65]: After ECMAStore.Build(), you can use OPSLogger.ErrorLogged to check if there's any error logged. If there's error, you can directly quit.

File: ECMA2Yaml/ECMAHelper/Models/ECMAStore.cs [hunk 2]
```diff
@@ -962,11 +962,18 @@ private void BuildInheritanceDefault(Type t)
...
+                                        // Create another HashSet since all the inheritedMembersById share the same "Monikers" object
+                                        inheritedMember.Monikers = inheritedMember.Monikers.Except(m.Monikers).ToHashSet();
+                                        if(inheritedMember.Monikers.Count == 0)
+                                        {
+                                            inheritedMembersById.Remove(m.Id);
+                                        }
+                                    }
+                                    else
+                                    {
+                                        OPSLogger.LogUserWarning(LogCode.ECMA2Yaml_Member_EmptyMoniker, m.SourceFileLocalPath, m.DocId);
```
Comment (TianqiZhang) [inline ECMA2Yaml/ECMAHelper/Models/ECMAStore.cs]: should be error

Comment (luyajun0205) [reply ECMA2Yaml/ECMAHelper/Models/ECMAStore.cs]: updated.

File: ECMA2Yaml/IntellisenseFileGen/IntellisenseFileGenHelper.cs (hunk 1 above)
Comment (luyajun0205) [reply ECMA2Yaml/IntellisenseFileGen/IntellisenseFileGenHelper.cs:65]: updated.

**Review by TianqiZhang — APPROVED** [review]

**Pull request merged**